feat: overhaul cards

jdrueckert opened the pull request:
- use bootstrap `Card` component instead of hand-made design
- add scss classes for card img and card subtitle
- differentiate between blogs and modules via `type`

4Denthusiast commented on _posts/2021-01-30-terasaturday-9.md:
```diff
@@ -38,7 +38,8 @@ _@Hajdam_ (with help from _@DarkWeird_) [fixed good ol' TeraEd for Linux and Win
 Both, _@4D Enthusiast_ and _@DarkWeird_ worked on various ways to improve performance 🐢🧙‍♂️
-_@4D Enthusiast_ started working on more chunk magic, in particular [rendering chunks in the far distance with less details](https://github.com/MovingBlocks/Terasology/pull/4427) (Level of Detail, LOD) in order to reduce the main memory consumption.
+_@4D Enthusiast_ worked on more chunk magic, in particular [rendering chunks in the far distance even before there's enough data to render them perfectly](https://github.com/MovingBlocks/Terasology/pull/4427) in order to reduce the main memory consumption.
+The next iteration in that area will target rendering them with less details (Level of Details, LOD).
```
Replace "rendering them" with "rendering even more distant chunks", then this phrasing is okay.

jdrueckert committed:
feat: overhaul cards

- use bootstrap `Card` component instead of hand-made design
- add scss classes for card img and card subtitle
- differentiate between blogs and modules via `type`

jdrueckert committed:
chore: fix prettier and eslint complaints

skaldarnar reviewed (approved):
We know `h7` is not a proper solution, will get back to this soon fixing up the styles.
